<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Система учёта веса</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainWindow.qrc">
    <normaloff>:/icons/icons/honeycomb.svg</normaloff>:/icons/icons/honeycomb.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	background-color: rgb(255, 255, 127);
    border: 1px solid rgb(0, 0, 0);
	border-radius:20px;
}

QPushButton::pressed{
	background-color: rgb(236, 226, 111);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnExport">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Экспорт</string>
    </property>
    <property name="icon">
     <iconset resource="mainWindow.qrc">
      <normaloff>:/icons/icons/export.svg</normaloff>:/icons/icons/export.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGraph">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>График</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="mainWindow.qrc">
      <normaloff>:/icons/icons/graph.svg</normaloff>:/icons/icons/graph.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNotebook">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Журнал</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="mainWindow.qrc">
      <normaloff>:/icons/icons/notebook.svg</normaloff>:/icons/icons/notebook.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSync">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>460</width>
      <height>350</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Синхронизировать</string>
    </property>
   </widget>
   <widget class="QWidget" name="placeData" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>130</width>
      <height>270</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>130</width>
      <height>270</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">.QWidget  {
	
	background-color: rgb(85, 255, 127);
	border: 2px solid rgb(0, 0, 0);
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="date">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid rgb(0, 0, 0);
background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="weight">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Нажмите, чтобы изменить сегодняшний вес</string>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Основная кнопка */
QPushButton#MainButton {
    border-radius: 8px;
    padding: 10px;
    font-size: 14pt;
    background-color: #f0f0f0;
}

/* Виджет-подсказка */
QLabel#TooltipLabel {
    visibility: hidden;
    position: absolute;
    top: 0;
    left: 100%;
    margin-left: 10px;
    background-color: rgba(0, 0, 0, 0.8); /* Полупрозрачный черный фон */
    color: white;
    padding: 5px;
    border-radius: 5px;
}

/* Когда наводится на основную кнопку, видим подсказку */
QPushButton#MainButton:hover ~ QLabel#TooltipLabel {
    visibility: visible;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnNewAccounting">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>490</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Создать новый учёт</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnNewAccounting{
	border-top-left-radius: 0px;
	border-bottom-left-radius: 0px;
	border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;
}

</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnList">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>490</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>Показать список учётов</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnList{
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
	border-top-right-radius: 0px;
	border-bottom-right-radius: 0px;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="mainWindow.qrc">
      <normaloff>:/icons/icons/menu.svg</normaloff>:/icons/icons/menu.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelList">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>490</y>
      <width>340</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#labelList {
	
	background-color: rgb(85, 255, 127);
	border: 1px solid rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Экран</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
